exercises/E-conditionals/exercise.js

```diff
@@ -9,6 +9,15 @@
 var name = "Daniel";
 var danielsRole = "mentor";
 
+if (name === "Daniel" && danielsRole === "mentor") {
+  console.log("Hi, I'm Daniel, I'm a mentor.");
+} else {
+  console.log("Hi, I'm " + `${name}` + " , I'm a student.");
```

Comment:
Still not the correct way to use string literal syntax, I will run through this with you sometime this week